pkgs/development/libraries/agda/generics/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, mkDerivation, fetchFromGitHub, standard-library }:
+
+mkDerivation rec {
+  pname = "generics";
+  version = "rc";
```

Comment:
It would be preferable to have a version here, but the project has no releases yet.

Reply:
I think the usual thing is to put the github revision in the version in these cases